### ISSUE TITLE
Using (modelChange) causes Illegal invocation when passing a file object v5.4.2 works with v4.4.2

### ISSUE THREAD

**Issue by 3vil4aron** [ngx-formly/ngx-formly#1788]
I'm updating to anglur 7 and ngx-formly 5 and now can't seem to pass my file object around. hope you can help me.

[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request
```

**Current behavior**
Using the input type="file" and a file value accessor on model chagnes is throwing a  Illegal invocation.

**Expected behavior**
Using the input type="file" and a file value accessor on model chagnes would return the file object 

**Minimal reproduction of the problem with instructions**
Here is repo:
https://stackblitz.com/edit/angular-ofbu2u

1. open file selector pick a file.
2. in console see the error.

Result: unable to pass the file to another component.

**What is the motivation / use case for changing the behavior?**
This is a regression from v4.4.2 

**Please tell us about your environment:**
windows 10, vscode, npm 10

* **Angular version:** 2.0.X
Angular version 7.2.15

* **Browser:**
all
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
All

* **Node (for AoT issues):** 
v10.16.0

**Comment by sanjaybhas** [comment]
Thank you @aitboudad for fixing this promptly. Could you please push out a new version with this fix?

**Comment by aitboudad** [comment]
@sanjaybhas do you think you can wait until the next week (next Thursday)? We'll release `5.5` version which includes this fix too.

**Comment by sanjaybhas** [comment]
@aitboudad, we have a critical dependency on this fix and need this asap since we already upgraded to the latest formly. Is there another workaround until the new version is officially published? Appreciate all your help.

**Comment by aitboudad** [comment]
ok, I'll push a fix release tomorrow!

**Comment by aitboudad** [comment]
released in `5.4.3` version.

**Comment by sanjaybhas** [comment]
Thank you very much.
